src/api/flaskr/service/study/output/ask.py

```
@@ -0,0 +1,53 @@
+    trace_args: dict,
+    trace: StatefulTraceClient,
+):
+
+    shifu_bid = outline_item_info.shifu_bid
+    app.logger.info(f"block_dto: {shifu_bid}")
+    follow_up_info = get_follow_up_info(
+        app, outline_item_info.shifu_bid, block_dto, attend_id
+    )
+
+    ask_mode = follow_up_info.ask_mode
+    visible = True if ask_mode == ASK_MODE_ENABLE else False
```

Ruff (0.12.2)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)